Shop-1/src/models/Products.ts

```diff
@@ -0,0 +1,21 @@
+import mongoose from "mongoose";
+
+const ProductsSchema = new mongoose.Schema(
+  {
+    name: {
+      type: String,
+      required: true,
+    },
+    price: {
+      type: Number,
+      required: true,
+    },
+    photoURL: {
+      type: String,
+    },
+  },
+  { timestamps: true }
+);
+
+const Product = mongoose.model("Product", ProductSchema);
```

Comment:
Match ProductSchema on line 20 to ProductSchema on line 3. one has an s, the other doesn't.